client/src/includes/initSkipLink.ts

```diff
@@ -0,0 +1,27 @@
+const initSkipLink = () => {
+  // Inspired by https://github.com/selfthinker/dokuwiki_template_writr/blob/master/js/skip-link-focus-fix.js
+
+  const skiplink = document.querySelector('[data-skiplink]');
+  const main = document.querySelector('main');
+
```

Comment:
@Julietadeboye  it looks like you have included changes from other work in this PR - you will need to create a new branch for the init skip link stuff.

I also recommend you create a new branch for this issue (tooltips one) and you may need to do a new PR.

Working off your `main` branch is going to make it very difficult to manage two PRs or more at once.

Reach out on the Outreachy channel if you need a hand creating different branches.

Reply:
oh my, didn't realise i was updating this PR  I'd correct this. Thanks